femmecubator/open-sprints
# Task/create mdx templates

#### What does this PR do?
- Creates MDX templates for commonly used elements
- Built Summary tab

#### How should this be manually tested?
http://localhost:3000/open-sprints/docs/projects/akita/
- Verify that functionality fulfills criteria in ticket - 
https://trello.com/c/ZzLU3ch1/14-us-akita001-project-page-markdown-templates-body-of-page

![image](https://github.com/femmecubator/open-sprints/assets/58869812/6fba008e-6cd1-4974-997b-e3bb9d5b368e)
